test/integration/auth.js

```diff
@@ -154,6 +184,89 @@ function test(utils) {
       });
   }
 
+  function testListUsers() {
+    var promises = [];
+    uids.forEach(function(uid) {
+      var tempUserData = {
+        uid: uid,
+        password: 'password'
+      };
+      promises.push(admin.auth().createUser(tempUserData));
+    });
+    return Promise.all(promises)
+      .then(function() {
+        // Return 2 users with the provided page token.
+        // This test will fail if other users are created in between.
+        return admin.auth().listUsers(2, uids[0]);
+      })
+      .then(function(listUsersResult) {
+        // Confirm expected number of users.
+        utils.assert(
+          listUsersResult.users.length == 2,
+          'auth().listUsers() expected number of results',
+          'Incorrect number of results.');
+        // Confirm next page token present.
+        utils.assert(
+          typeof listUsersResult.pageToken === 'string',
```

Comment:
Just wondering how does this method signal end of user list. Perhaps document that at the method definition.

Reply:
Added to listUsers description.